src/vcSettingsUI.cpp

```diff
@@ -566,8 +568,10 @@ void vcSettingsUI_Show(vcState *pProgramState)
             ImGui::Indent();
             if (ImGui::InputInt2(vcString::Get("settingsScreenshotResLabel"), (int*)&pProgramState->settings.screenshot.resolution.x))
             {
-              pProgramState->settings.screenshot.resolution.x = udClamp(pProgramState->settings.screenshot.resolution.x, 32u, 16384u);
-              pProgramState->settings.screenshot.resolution.y = udClamp(pProgramState->settings.screenshot.resolution.y, 32u, 16384u);
+              pProgramState->settings.screenshot.resolution.x = udClamp(pProgramState->settings.screenshot.resolution.x
+                , ScreenshotResolutionsMin.x, ScreenshotResolutionsMax.x);
+              pProgramState->settings.screenshot.resolution.y = udClamp(pProgramState->settings.screenshot.resolution.y
+                , ScreenshotResolutionsMin.y, ScreenshotResolutionsMax.y);
```

Comment:
This isn't really how we do formatting, we don't really break things up onto multiple lines.

Also, I think you could probably change the above to:
```c++
pProgramState->settings.screenshot.resolution = udClamp(pProgramState->settings.screenshot.resolution, screenshotResolutionsMin, ScreenshotResolutionsMax);
```
We have overloads for `udClamp` and friends to handle `udVectorX<T>`, we may as well utilise them.

Reply:
Resolved.